src/snowflake/connector/version.py

```diff
@@ -1,3 +1,3 @@
 # Update this for the versions
 # Don't change the forth version number from None
-VERSION = (2, 9, 0, None)
+VERSION = (2, 9, 2, None)
```

Comment:
Curious: Where is 2.9.1 😂

Reply:
this PR is expected to be merged after release of 2.9.1